general/clients/clients.md

```diff
@@ -135,15 +135,15 @@ The official Jellyfin iOS app.
 
 The official Jellyfin Kodi plugin.
 
-**Status:** ⭐ Receiving patches
+**Status:** ⭐ Active
 
 **Links:**
 
 * [GitHub](https://github.com/jellyfin/jellyfin-kodi)
 
 #### Installing
 
-1. Download the repository installer [here](https://jellyfin.org/downloads/clients/#kodi-repository)
+1. Download the repository installer [here](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip)
```

Comment:
maybe we should just link to https://repo.jellyfin.org/releases/client/kodi

Reply:
My thought was since we say "It'll download as ____", it's implying that is the actual download link.  Linking it to `/kodi` basically just adds another click for users.  Plus that download page is super ugly and unstyled after how nice the rest of the site looks